integration/control_plane_test.go

```diff
@@ -45,23 +45,6 @@ func TestDashboard(t *testing.T) {
 		}
 		defer res.Body.Close()
 
```

Comment:
The integration test is still there. This is removed because it's outdated integration test. Now ForwardAuth is separated service, we don't have user dashboard for it, the same as authorize service.